app/models/comment.rb

```diff
@@ -4,6 +4,9 @@ class Comment < ApplicationRecord
 
   after_save :update_comments_counter
 
+  validates :author_id, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :post_id, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
   def update_comments_counter
     post.increment!(:comments_counter)
   end
```

Comment:
- [ ] When you post a comment you should display a flash message as well please work on that

Reply:
- [ ] That should be applied on the `likes` as well